### DISCUSSION TITLE
The walletconnect session information is being stored in indexedDB, how can I save it to localStorage?

### DISCUSSION THREAD

**Discussion by kiles** [WalletConnect/walletconnect-monorepo#4092] [Developer Support] [answered]
hello.

I was implementing the wallet connection feature with WalletConnect V2 using UniversalProvider.

Since a few days ago, I noticed that when I try to connect with WalletConnect, the wc:// related information that was stored in LocalStorage is now stored in IndexedDB.

Is there any way to set it to be stored in LocalStorage instead of IndexedDB?

**Comment by glitch-txs** [comment]
Hi, what's the use case?

**Reply by kiles** [reply]
I just init and use it like below, but there is no other setting, but suddenly it is not saved in localstorage, but in indexedDB.


```
const provider = await UniversalProvider.init({
      projectId: "",
      logger: "",
      relayUrl: "wss://relay.walletconnect.com",
      metadata: {
        name: "",
        description: "Description",
        url: "https://....",
        icons: ["https://..."],
      },
    });
```

**Reply by glitch-txs** [reply]
yes, why would you like it to be stored in localStorage instead?

**Reply by kiles** [reply]
I'm having an issue with certain mobile devices not saving values to IndexedDB.

**Reply by glitch-txs** [reply, marked as answer]
which devices? could you share more information and how we can reproduce this issue?

**Reply by kiles** [reply]
I wasn't able to figure out the exact cause, but once I got the indexedDB to initialize, the values were generated normally.
Thanks for your feedback.
